common/content/finder.js

```diff
@@ -124,15 +128,27 @@ var Finder = Module("finder", {
 
             // PDF.JS files are different, they use their own messages to communicate the results.
             // So we piggyback the end changes to the findbar when there are any.
-            findbar._original_updateControlState = findbar.updateControlState;
+            findbar._vimpbackup_updateControlState = findbar.updateControlState;
             findbar.updateControlState = function(aResult, aFindPrevious) {
-                this._original_updateControlState(aResult, aFindPrevious);
+                this._vimpbackup_updateControlState(aResult, aFindPrevious);
                 finder.onFindResult({
                     searchString: this._findField.value,
                     result: aResult,
                     findBackwards: aFindPrevious
                 });
             };
+
+            // Normally the findbar appears to notify on failed results.
+            // However, this shouldn't happen when we're finding through the command line,
+            // even though that way we lose any kind of no matches notification until we
+            // stop typing altogether; something to work on at a later time:
+            // - show the quick findbar which will hide after a few seconds?
+            // - or notify the user somehow in the command line itself?
```

Comment:
yep, we should have the commandline turn red or something to signal this. We can either work on it as part of this PR or we can merge this one in and then work on it in another one.